src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable {
```

Comment:
if every class that extends Figure implements Drawable, can we implement Drawable from Figure?